automatic/webstorm/tools/ChocolateyUnInstall.ps1

```diff
@@ -0,0 +1,37 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$packageName = 'WebStorm'
+$softwareName = 'JetBrains WebStorm*'
+$installerType = 'exe'
+
+$silentArgs = '/S'
+$validExitCodes = @(0, 3010, 1605, 1614, 1641)
+
+if ($installerType -ne 'MSI') {
+    $validExitCodes = @(0)
```

Comment:
Instead of overriding the validExitCodes here, change the validExitCode a few lines up to only include `@(0)` and remove the if statement.